source/hosts-resolver.js

```diff
@@ -1,20 +1,20 @@
 'use strict';
+const path = require('path');
 const {watchFile} = require('fs');
 const {readFile} = require('fs').promises;
 const {isIP} = require('net');
 
 const isWindows = process.platform === 'win32';
-const hostsPath = isWindows ? `${process.env.SystemDrive}\\Windows\\System32\\drivers\\etc\\hosts` : '/etc/hosts';
+const hostsPath = isWindows ? path.join(process.env.SystemDrive, 'Windows\\System32\\drivers\\etc\\hosts') : '/etc/hosts';
```

Comment:
It's safer this way. Maybe someone accidentally ended the path in `SystemDrive` with a slash.